docs/docs/forms.mdx

```diff
@@ -251,18 +252,19 @@ rules:
 
 In some situations, users may change their mind in the middle of the form action
 and decide not to go forward with their initial request. In cases like this, the
-assistant should stop asking for the requested slots. You can handle such situations
+assistant should stop asking for the requested slots.
+
+You can handle such situations
 gracefully using a default action `action_deactivate_loop` which will deactivate
 the form and reset the requested slot. An example story of such conversation could
 look as follows:
 
 ```yaml-rasa
-rules:
-- rule: Example of an unhappy path
-  condition:
-  # Condition that form is active.
-  - active_loop: restaurant_form
+- story: User interrupts the form and doesn't want to continue
   steps:
+  - intent: request_restaurant
+  - action: restaurant_form
+  - active_loop: restaurant_form
```

Comment:
the problem with this story is that it'll not work if there was `chitchat` before `stop`

Reply:
I think that's fine, here we're only trying to show what to do if the user wants to stop, not handle all cases